src/config/index.js

```diff
@@ -4,6 +4,7 @@ exports.COMMAND = {
   open: 'open',
   config: 'config',
   build: 'build',
+  output: 'output',
```

Comment:
I think the command `output` is a bit generic in this scenario. I was thinking, to name this command as a `console`, that provides a bit more context and perfectly matches with the alias `co`. LMK WDYT?